libs/klv/src/main/java/org/codice/alliance/libs/klv/Utilities.java

```diff
@@ -25,4 +31,30 @@ public static boolean isEmptyString(Serializable serializable) {
     public static boolean isNotEmptyString(Serializable serializable) {
         return !isEmptyString(serializable);
     }
+
+    static void safelySetAttribute(Metacard metacard, Attribute attribute) {
+        safelySetAttribute(metacard, attribute.getName(), attribute.getValues());
```

Comment:
without knowing all the places this is used, can we really know that metacard and attribute won't be null?